src/App.tsx

```diff
@@ -20,21 +20,21 @@ class Counter extends React.Component<CounterProps> {
     this.decrement = this.decrement.bind(this);
   }
 
-  increment() {
+  increment(): void {
     this.setState({ value: this.state.value + 1 });
   }
 
-  decrement() {
+  decrement(): void {
     this.setState({ value: this.state.value - 1 });
   }
 
-  render() {
+  render(): ReactElement {
```

Comment:
I'd say this is too much typing. Typing is great but this is overzealous :)
If your lint config forces you to type absolutely everything then it's a bad config. If this was your own decision then you can go for it but I see no benefit in such obvious types.

Reply:
I removed all the unusual suggestions, including this one. thank you